java-frontend/src/main/java/org/sonar/plugins/java/api/JavaResourceLocator.java

```diff
@@ -53,14 +53,29 @@ public interface JavaResourceLocator extends JavaFileScanner {
   /**
    * The folders containing the binary .class files.
    * @return a list of folders.
+   * @since 9.7
    */
   Collection<File> binaryDirs();
 
+  /**
+   * The folders containing the binary .class files corresponding to the tests.
+   * @return a list of folders.
+   * @since 9.7
+   */
+  Collection<File> testBinaryDirs();
+
   /**
    * Classpath configured for the project.
    * This classpath method is used by the findbugs plugin to configure the analysis.
    * @return the list of jar and class files constituting the classpath of the analyzed project.
    */
   Collection<File> classpath();
 
+  /**
+   * Classpath configured for the project tests.
+   * This classpath method is used by the findbugs plugin to configure the analysis.
```

Comment:
I would avoid explicitly mentioning dependent projects that we don't maintain. I will create a JIRA ticket with a summary mentioning findbugs and we can mention the ticket in the final merge commit.
```suggestion
```

Reply:
Agreed, I've removed that mention to findbugs as well as the one on `classpath()`
I suppose it dates from the time when that plugin used to be managed by SonarSource

I've also adjusted the `@Since` to the next version of SonarQube, presumably 9.8 ?

For the ticket and the changelog, is that something I should be taking care of?

Reply:
No worries, I will take care of the ticket. Thank you for the  quick changes